### PR TITLE
Fix #33

### LINKED ISSUE #33: Minor Code Style Thing
https://github.com/phoboslab/Impact/blob/f461c2f4054dac37dea351d8fe3440eae6a2f47e/lib/weltmeister/weltmeister.js#L466

There should probably be a space after the `:` to match the style of the lines above it.

### PR DESCRIPTION
**Pull request by therocco**

### CODE REVIEW

**Comment by Joncom** [comment]
Possible to do it without making all those whitespace changes unrelated to the fix?

**Comment by therocco** [comment]
updated :+1:

File: lib/weltmeister/weltmeister.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Joncom) [inline lib/weltmeister/weltmeister.js]: One left here?

**Comment by Joncom** [comment]
Would it be possible to squash it all into a single commit so that adding literally just a single space doesn't require multiple commits?

**Pull request closed without merging**